frontend/src/components/mine/Tailings/FileUpload.js

```diff
@@ -0,0 +1,52 @@
+import { map } from "lodash";
+import React from "react";
+import { FilePond, File, registerPlugin } from "react-filepond";
+import "filepond/dist/filepond.min.css";
+import FilePondPluginFileValidateSize from "filepond-plugin-file-validate-size";
+registerPlugin(FilePondPluginFileValidateSize);
+
+import { ENVIRONMENT } from "@/constants/environment";
+import { createRequestHeader } from "@/utils/RequestHeaders";
+
+class FileUpload extends React.Component {
+  constructor(props) {
+    super(props);
+
```

Comment:
```
constructor(props) {
    super(props);
```

This can be removed. It is not serving any purpose. 

In older versions of react the constructor was needed and the state was initiated inside, like so: 
```
constructor(props) {
 super(props);
 this.state = { files: [] };
}
```

But state can be initiated on its own without the constructor, which is what you are doing on line 15, (also consistent with the other files)

so line 12 & 13 are not actually doing anything.

Reply:
After discussion, this was a miss-understanding that the server parameter was set inside the constructor not outside of it. Marking as resolved.